nion/swift/ImageCanvasItem.py

```diff
@@ -304,7 +340,28 @@ def __init__(self, screen_pixel_per_image_pixel_stream: Stream.ValueStream[float
         self.__screen_pixel_per_image_pixel_stream = screen_pixel_per_image_pixel_stream.add_ref()
         self.__screen_pixel_per_image_pixel_action = Stream.ValueStreamAction(screen_pixel_per_image_pixel_stream, lambda x: self.__update_sizing())
         self.__scale_marker_width = 0
+        self._scale_marker_position: typing.Optional[str] = None
+        self._background_fill_color: typing.Optional[str] = None
```

Comment:
Should these be private variables (i.e. two underscores)?

Also, can you use `str | None` style typing for new code?

```
        self.__scale_marker_position: str | None = None
        self.__background_fill_color: str | None = None
```